docs/spec/sanitations.md

```diff
@@ -1,24 +1,24 @@
-_Author_:  <!-- TODO: Add author name --> \
-_Created_: <!-- TODO: Add date --> \
-_Updated_: <!-- TODO: Add date --> \
+_Author_:  @Sachindu-Nethmin\
+_Created_: 2024/10/22\
+_Updated_: 2024/10/22\
 _Edition_: Swan Lake
 
 # Sanitation for OpenAPI specification
 
 This document records the sanitation done on top of the official OpenAPI specification from Spotify. 
-The OpenAPI specification is obtained from (TODO: Add source link).
+The OpenAPI specification is obtained from the [Spotify OpenAPI Documentation](https://developer.spotify.com/reference/web-api/open-api-schema.yaml).
 These changes are done in order to improve the overall usability, and as workarounds for some known language limitations.
 
 [//]: # (TODO: Add sanitation details)
-1. 
-2. 
-3. 
+1. Remove resource paths,
+    */playlists/{playlist_id}/images'(put) - schema for the image/jpeg content type is incorrectly defined
```

Comment:
@Sachindu-Nethmin can you elaborate more on this?

Reply:
@NipunaRanasinghe 
  In '/playlists/{playlist_id}/images'
    Remove the required Field from the Schema because the required attribute within the schema for the image/jpeg content type is incorrectly defined. I think I did it in an alternative way now.